src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1574,7 +1574,7 @@ public async Task ValidateLocationHeaderFieldForPostRequests(EntitySourceType en
             HostOptions staticWebAppsHostOptions = new(null, AuthenticationOptions);
 
             RuntimeOptions runtimeOptions = configuration.Runtime;
-            RuntimeOptions baseRouteEnabledRuntimeOptions = new(runtimeOptions.Rest, runtimeOptions.GraphQL, staticWebAppsHostOptions, "/data-api");
+            RuntimeOptions baseRouteEnabledRuntimeOptions = new(runtimeOptions?.Rest, runtimeOptions?.GraphQL, staticWebAppsHostOptions, "/data-api");
```

Comment:
nit: the above line `configuration.Runtime` returns a nullable `RuntimeOptions`. so `RuntimeOptions runtimeOptions` should be `RuntimeOptions? runtimeOptions`

Reply:
the test project doesnt have "nullable" context enabled. So, I couldnt do the suggested. Enabling nullable context for test project will be a refactoring issue in itself.

It errors:

> Severity	Code	Description	Project	File	Line	Suppression State
> Error	CS8632	The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.		1576	Active